### PR TITLE
[DOCS-2866] Updating CCM documentation

### PR DESCRIPTION
**Pull request by sunhay**
### Preview

https://docs-staging.datadoghq.com/sunhay/ccm-updates/infrastructure/cloud_cost_management/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/infrastructure/cloud_cost_management.md [hunk 1]
```diff
@@ -85,7 +95,23 @@ Attach the new S3 policy to the Datadog integration role.
...
+
+## Cloud costs in dashboards
+
+Visualizing infrastructure spend alongside related utilization metrics can help you spot potential inefficiencies and savings opportunities. You can add cloud costs to widgets in Datadog dashboards by selecting the *Cloud Cost* data source.
  
+{{< img src="infrastructure/cloudcost/cloud_cost_data_source.png" alt="Cloud Cost available as a data source in dashboard widget creation"  >}}
+
 ## Cost types
  
 You can visualize your ingested data using the following cost types:
```
Comment (sunhay) [inline content/en/infrastructure/cloud_cost_management.md:117]: TODO: Need more detailed metric descriptions and easy to understand examples

Comment (sunhay) [reply content/en/infrastructure/cloud_cost_management.md:117]: Perhaps we can just quote the examples linked here? https://aws.amazon.com/blogs/aws-cloud-financial-management/understanding-your-aws-cost-datasets-a-cheat-sheet/

File: content/en/infrastructure/cloud_cost_management.md [hunk 2]
```diff
@@ -16,6 +16,16 @@ To use Cloud Cost Management, you must have an AWS account with access to Cost U
...
+### Configuration
+
+To configure a data source, access the Cloud Costs product by clicking **Infrastructure > Cloud Costs** using the main navigation.
+
+If there are no configured data sources, you are redirected to a setup page with instructions for generating and enabling access to your AWS account's [Cost and Usage Report (CUR)][5].
+
+{{< img src="infrastructure/cloudcost/cloud_cost_data_source_config.png" alt="Configure the AWS data source for Cloud Costs"  >}}
+
+Once configured, data sources may be viewed and modified by accessing the **Settings tab** at the top of the page.
+
```
Comment (ocr) [inline content/en/infrastructure/cloud_cost_management.md:28]: Might be useful here to mention that we currently support only a single data source, i.e. a single account at a time (and maybe also remind the reader that for now its only AWS).

File: content/en/infrastructure/cloud_cost_management.md (hunk 1 above)
Comment (ocr) [reply content/en/infrastructure/cloud_cost_management.md:117]: I'd definitely recommend adding that link

File: content/en/infrastructure/cloud_cost_management.md [hunk 3]
```diff
@@ -85,7 +95,23 @@ Attach the new S3 policy to the Datadog integration role.
...
 4. Enter the name of the S3 bucket policy created above.
 5. Click **Attach policy**.
+
+## Investigating your costs
+
+Navigate to your costs within Datadog by clicking **Infrastructure > Cloud Costs** in the UI. This view lets you quickly filter and group your spending to hone in on what’s changing over time, answering questions such as:
+
+- Which teams and applications are responsible for changes in spend?
+- What cloud products and usage types (i.e. EC2 Data Transfer) are driving the increase in spend?
+- How effective is the coverage and utilization of my discount plans?
```
Comment (ocr) [inline content/en/infrastructure/cloud_cost_management.md:105]: Could we maybe add examples here?
Something like this:
Which teams and applications are responsible for changes in spend? Try grouping by service and team.

Also, not so sure about discount plans. How are we doing this?

File: content/en/infrastructure/cloud_cost_management.md [hunk 4]
```diff
@@ -85,7 +95,23 @@ Attach the new S3 policy to the Datadog integration role.
...
+
+{{< img src="infrastructure/cloudcost/cloud_cost_analyze_page.png" alt="Slice and dice your costs by tag using the Cloud Cost UI"  >}}
+
+## Cloud costs in dashboards
+
+Visualizing infrastructure spend alongside related utilization metrics can help you spot potential inefficiencies and savings opportunities. You can add cloud costs to widgets in Datadog dashboards by selecting the *Cloud Cost* data source.
```
Comment (ocr) [inline content/en/infrastructure/cloud_cost_management.md:111]: Would we have an example using F&F that we could show here?

File: content/en/infrastructure/cloud_cost_management.md [hunk 5]
```diff
@@ -30,7 +40,7 @@ Select the following Delivery options:
 * Compression type: **GZIP**
 * Format: `text/csv`
  
-### Configure access to the CUR
+### Allow access to the CUR
  
 Configure AWS to ensure Datadog has permissions to access the CUR and the s3 bucket it is stored in by [creating a policy][2] using the following JSON:
```
Comment (cnharris10) [inline content/en/infrastructure/cloud_cost_management.md:45]: Nit: `s3` --> `S3`

File: content/en/infrastructure/cloud_cost_management.md (hunk 3 above)
Comment (cnharris10) [reply content/en/infrastructure/cloud_cost_management.md:105]: We have the foundation set up for discount plan coverage/utilization, but we are not exposing this yet

**Comment by apigirl** [comment]
hello! closing this because it is stale, please feel free to reactivate if you have the bandwidth to work on it!

**Pull request closed without merging**